pkg/api/helpers.go

```diff
@@ -529,8 +536,23 @@ func GetAvoidPodsFromNodeAnnotations(annotations map[string]string) (AvoidPods,
 	return avoidPods, nil
 }
 
-// SysctlsFromPodAnnotation parses an annotation value of the key SysctlsPodAnnotationKey
-// into a slice of Sysctls.
+// SysctlsFromPodAnnotation parses the sysctl annotations a slice of safe Sysctls
```

Comment:
s/SysctlsFromPodAnnotation/SysctlsFromPodAnnotations/. Also, I think some words are missing - "parses the sysctl annotations a slice of safe Sysctls ..."?